src/Misc.php

```diff
@@ -50,6 +50,11 @@ public static function absolutize_url($relative, $base)
         return $iri->get_uri();
     }
 
+    public static function is_remote_uri(string $uri): bool
```

Comment:
We should add a note at the `CHANGELOG.md` about this new method.

Reply:
Good point. I have decided to mark it as internal instead to avoid increasing the API surface and having to deprecate it later.